app/pages/project/home/index.jsx

```diff
@@ -91,9 +91,16 @@ export default class ProjectHomeContainer extends React.Component {
   }
 
   render() {
+    let shownWorkflows = [];
+    if (this.props.project.experimental_tools.includes('hide completed workflows')) {
+      shownWorkflows = this.state.activeWorkflows.filter(workflow => workflow.completeness !== 1);
```

Comment:
Filtering this array on every call to render is very inefficient. I think it would be much faster to filter the array once, and store the filtered array in component state.